### PR TITLE
add gitpod config

### PR DESCRIPTION
**Pull request by flaviuvadan**
Fixes: #333 
CC: @samj1912

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 The Argo was constructed by the shipwright Argus,
 and its crew were specially protected by the goddess Hera.
 ```
-
 (https://en.wikipedia.org/wiki/Argo)
 
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#<https://github.com/argoproj-labs/hera-workflows>)
```
Comment (samj1912) [inline README.md:10]:
```suggestion
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/argoproj-labs/hera-workflows)
```

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+tasks:
+  - init: |
+      python -m pip install --upgrade pip
+      python -m pip install "tox<4.0" poetry
+      poetry install
```
Comment (samj1912) [inline .gitpod.yml:5]: Do we also want to include the python test explorer plugin? https://www.gitpod.io/docs/introduction/languages/python#python-test-explorer

**Comment by samj1912** [comment]
Thanks for this. In the past this has been very helpful for some other projects I have been a part of in terms of attracting new contributors.

File: .gitpod.yml (hunk 2 above)
Comment (flaviuvadan) [reply .gitpod.yml:5]: Will do!

**Pull request merged**